test/suites/gvcf/resources/liftover.bed

```diff
@@ -0,0 +1 @@
+22	0	42127526
```

Comment:
better to test behavior of zero-based `[0, 42127525)` here for `22	42127525	.	A	AT	3.9	PASS	.	GT:GQ:DP:AD:VAF:PL	1/1:2:2:0,2:1:0,4,0`?

Reply:
to test zero based properly it should 42127524 right?